diff --git a/node_modules/accessible-astro-components/AccordionItem.astro b/node_modules/accessible-astro-components/AccordionItem.astro
index 468b68a..305fc2a 100644
--- a/node_modules/accessible-astro-components/AccordionItem.astro
+++ b/node_modules/accessible-astro-components/AccordionItem.astro
@@ -1,5 +1,5 @@
 ---
-const {header} = Astro.props
+const {date, header} = Astro.props
 ---
 
 <li class="accordion__item">
@@ -11,7 +11,10 @@ const {header} = Astro.props
       aria-controls="accordion-panel-1"
       onclick="toggleAccordionItem()"
     >
-      {header}
+    <div class="accordion__date-title__container">
+      <span class="accordion__header__title">{header}</span>
+      <span class="accordion__header__date">{date}</span>
+    </div>
       <svg class="header__toggle-indicator" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
         <path fill="currentColor" d="M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z"/>
       </svg>
@@ -40,7 +43,35 @@ const {header} = Astro.props
     justify-content: space-between;
     border: 2px solid #202020;
     padding: 1rem;
+    transition: all .4s ease-in-out;
+  }
+
+  .accordion__header__title{
+    font-size: 1.5rem;
+    color: #002ea5;
+    font-weight: 600;
+  }
+  .accordion__header__date{
+    font-size: 1.2rem;
+    color: #002ea5;
+    min-width: 200px;
+  }
+  .accordion__item.is-active .accordion__header .accordion__header__title,
+  .accordion__item.is-active .accordion__header .accordion__header__date{
+    color: white;
   }
+  .accordion__date-title__container{
+    display: flex;
+    flex-direction: column;
+    margin-right: 1rem;
+  }
+  @media screen and (max-width: 600px) {
+    .accordion__date-title__container{
+      flex-direction: column;
+      row-gap: .5em;
+    }
+  }
+
   
   .accordion__item .accordion__header {
     border-bottom: 0;
@@ -62,7 +93,8 @@ const {header} = Astro.props
   }
 
   .header__toggle-indicator {
-    width: 20px;
+    min-width: 20px;
+    max-width: 20px;
   }
 
   .accordion__panel {
